packages/patternfly-4/react-core/src/components/Modal/examples/Modal.md

```diff
@@ -62,6 +64,58 @@ class SimpleModal extends React.Component {
 }
 ```
 
+```js title=Basic-with-description
```

Comment:
Change to just "With description" to be consistent with https://pf4.patternfly.org/documentation/core/components/modalbox#with-description